build_locally.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash -eu
 # Locally build the Jekyl site for testing on Ubuntu 16.04
 
+have_travis() {
```

Comment:
This is an extremely verbose function, which is not commented ;-)
Using my single-line expression, it becomes obsolete.

Reply:
yes but it supports multi-arguments, so its necessary